src/widget/gui.cpp

```diff
@@ -195,46 +163,38 @@ QString GUI::itemInputDialog(QWidget * parent, const QString & title,
                     Qt::InputMethodHints hints)
 {
     if (QThread::currentThread() == qApp->thread())
-    {
         return getInstance()._itemInputDialog(parent, title, label, items, current, editable, ok, flags, hints);
-    }
-    else
-    {
-        QString r;
-        QMetaObject::invokeMethod(&getInstance(), "_itemInputDialog", Qt::BlockingQueuedConnection,
-                    Q_RETURN_ARG(QString, r),
-                    Q_ARG(QWidget*, parent), Q_ARG(const QString&, title),
-                    Q_ARG(const QString&,label), Q_ARG(const QStringList&, items),
-                    Q_ARG(int, current), Q_ARG(bool, editable), Q_ARG(bool*, ok),
-                    Q_ARG(Qt::WindowFlags, flags), Q_ARG(Qt::InputMethodHints, hints));
-        return r;
-    }
+
+    QString r;
+    QMetaObject::invokeMethod(&getInstance(), "_itemInputDialog", Qt::BlockingQueuedConnection,
+                Q_RETURN_ARG(QString, r),
+                Q_ARG(QWidget*, parent), Q_ARG(const QString&, title),
+                Q_ARG(const QString&,label), Q_ARG(const QStringList&, items),
+                Q_ARG(int, current), Q_ARG(bool, editable), Q_ARG(bool*, ok),
+                Q_ARG(Qt::WindowFlags, flags), Q_ARG(Qt::InputMethodHints, hints));
+    return r;
 }
 
 QString GUI::passwordDialog(const QString& cancel, const QString& body)
 {
     if (QThread::currentThread() == qApp->thread())
-    {
         return getInstance()._passwordDialog(cancel, body);
-    }
-    else
-    {
-        QString r;
-        QMetaObject::invokeMethod(&getInstance(), "_passwordDialog", Qt::BlockingQueuedConnection,
-                        Q_RETURN_ARG(QString, r),
-                        Q_ARG(const QString&, cancel), Q_ARG(const QString&, body));
-        return r;
-    }
+
+    QString r;
+    QMetaObject::invokeMethod(&getInstance(), "_passwordDialog", Qt::BlockingQueuedConnection,
+                    Q_RETURN_ARG(QString, r),
+                    Q_ARG(const QString&, cancel), Q_ARG(const QString&, body));
+    return r;
 }
 
 // Private implementations
 
-void GUI::_clearContacts()
+void GUI::_clearContacts() const
 {
     Nexus::getDesktopGUI()->clearContactsList();
 }
 
-void GUI::_setEnabled(bool state)
+void GUI::_setEnabled(bool state) const
```

Comment:
This and the functions around modify the GUI, they can't be const either, even if they don't directly touch private variables.